lib/improver/tests/cube_combiner/test_CubeCombiner.py

```diff
@@ -76,242 +72,134 @@ def test_basic(self):
-        expected_data = np.zeros((1, 2, 2, 2))
-        expected_data[0, 0, :, :] = 0.1
-        expected_data[0, 1, :, :] = 0.6
+        result = plugin.combine(self.cube1, self.cube2)
+        expected_data = np.full((1, 2, 2), 0.5, dtype=np.float32)
         self.assertArrayAlmostEqual(result.data, expected_data)
 
     def test_mean(self):
         """Test that the function adds the cubes correctly for mean."""
         operation = '+'
         plugin = CubeCombiner(operation)
-        result = plugin.combine(self.cube1, self.cube3,
-                                operation)
-        expected_data = np.zeros((1, 2, 2, 2))
-        expected_data[0, 0, :, :] = 0.6
-        expected_data[0, 1, :, :] = 1.4
+        result = plugin.combine(self.cube1, self.cube2)
+        expected_data = np.full((1, 2, 2), 1.1, dtype=np.float32)
         self.assertArrayAlmostEqual(result.data, expected_data)
 
 
-class Test_process(IrisTest):
+class Test_process(set_up_cubes):
 
     """Test the plugin combines the cubelist into a cube."""
 
-    def setUp(self):
-        """ Set up cubes for testing. """
-        self.cube1 = create_cube_with_threshold()
-        data = np.zeros((1, 2, 2, 2), dtype=np.float32)
-        data[0, 0, :, :] = 0.1
-        data[0, 1, :, :] = 0.4
-        self.cube2 = create_cube_with_threshold(data=data)
-        data2 = np.zeros((1, 2, 2, 2), dtype=np.float32)
-        data2[0, 0, :, :] = 0.9
-        data2[0, 1, :, :] = 0.2
-        self.cube3 = create_cube_with_threshold(data=data2)
-
     def test_basic(self):
         """Test that the plugin returns a Cube. """
         plugin = CubeCombiner('+')
-        cubelist = iris.cube.CubeList([self.cube1, self.cube1])
+        cubelist = iris.cube.CubeList([self.cube1, self.cube2])
         result = plugin.process(cubelist, 'new_cube_name')
         self.assertIsInstance(result, Cube)
         self.assertEqual(result.name(), 'new_cube_name')
-        expected_data = np.zeros((1, 2, 2, 2))
-        expected_data[:, 0, :, :] = 1.0
-        expected_data[:, 1, :, :] = 1.2
+        expected_data = np.full((1, 2, 2), 1.1, dtype=np.float32)
         self.assertArrayAlmostEqual(result.data, expected_data)
 
     def test_mean(self):
         """Test that the plugin calculates the mean correctly. """
-        plugin = CubeCombiner('mean')
+        plugin = CubeCombiner('add')
         cubelist = iris.cube.CubeList([self.cube1, self.cube2])
         result = plugin.process(cubelist, 'new_cube_name')
-        expected_data = np.zeros((1, 2, 2, 2))
-        expected_data[:, 0, :, :] = 0.3
-        expected_data[:, 1, :, :] = 0.5
+        expected_data = np.full((1, 2, 2), 1.1, dtype=np.float32)
+        self.assertEqual(result.name(), 'new_cube_name')
+        self.assertArrayAlmostEqual(result.data, expected_data)
+
+    def test_bounds_expansion(self):
+        """Test that the plugin calculates the sum of the input cubes
+        correctly and expands the requested coordinate bounds in the
```

Comment:
Got this docstring muddled with the above - this one calculates the mean, above calculates the sum.

Reply:
I meant this to be a sum, I accidentally changed the test above to "add" and this one to "mean", I've switched these and set the expected values correctly.